src/main.rs

```diff
@@ -13,11 +13,25 @@ struct Cli {
 #[derive(Subcommand)]
 enum Commands {
     /// Init a new App Chain config
-    Init,
+    Init {
+        /// App chain name
+        #[arg(short, long = "chain-name")]
+        name: Option<String>,
+        /// Select Sovereign
+        #[arg(short, long = "chain-mode")]
+        mode: Option<String>,
+        /// Select between Avail, Celestia, Ethereum, NoDA
```

Comment:
let the comment be select da layer, otherwise you need to change on every new addition

Reply:
found a way to pass Enum itself, updated